hip_data_tools/google/sheets.py

```diff
@@ -0,0 +1,82 @@
+import gspread
+
+
+class SheetUtil:
+    """
+    Utility class for connecting to google sheets
+    Args:
+        credentials (ServiceAccountCredentials): credentials to read google sheets
+    """
+
+    def __init__(self, credentials):
+        self.gc = gspread.authorize(credentials)
+
+    def get_value_matrix(self, workbook_name, sheet_name, cell_range='', skip_top_rows_count=0):
+        """
+        Get the values of the sheet as a 2D array
+        Args:
+            workbook_name (string): name of the workbook
+            sheet_name (string): name of the sheet
+            cell_range (string): range of the cells that need to be selected
+            skip_top_rows_count (integer):
+        Returns: values of the sheet as a 2D array
+
+        """
+        worksheet = self.gc.open(workbook_name).worksheet(sheet_name)
+        list_of_lists = []
+
+        if not cell_range:
+            list_of_lists = worksheet.get_all_values()
+            del list_of_lists[0:skip_top_rows_count]
+            return list_of_lists
+        else:
+            for cell in worksheet.range(cell_range):
+                list_of_lists.append(cell.value)
+            return list_of_lists
+
+    def get_table_settings(self, table_name, field_names, s3_bucket, s3_dir):
```

Comment:
I think this should not belong here.
the sheet util should manage reading and writing to / from sheet from / to standard formats like dict and dataframe
Athen util has the logic to write to athena from dict and pandas df

Reply:
Moved this method to google_sheet_to_athena etl class